analyzers/tests/SonarAnalyzer.UnitTest/Extensions/ITupleOperationWrapperExtensionsTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.CodeAnalysis.CSharp;
+using SonarAnalyzer.Extensions;
+using StyleCop.Analyzers.Lightup;
+
+namespace SonarAnalyzer.UnitTest.Extensions
+{
+    [TestClass]
+    public class ITupleOperationWrapperExtensionsTests
+    {
+        [DataTestMethod]
+        // Tuple expression on the right side of assignment
```

Comment:
[education] I don't get the behavior of the method - it's highly confusing to me a method `AllElements` which has an inconsistent behavior:
- if there is only one tuple (right or left), it returns all elements
- if there are two tuples (both right and left), it only returns **only** the ones on the left

I believe I'm a bit ignorant about something - could you please educate me?